src/langsmith/evaluation-concepts.mdx

```diff
@@ -3,238 +3,389 @@ title: Evaluation Concepts
 
-* [**Datasets**:](/langsmith/evaluation-concepts#datasets) Collections of test inputs and reference outputs.
-* [**Evaluators**](/langsmith/evaluation-concepts#evaluators): Functions for scoring outputs. These can be [online evaluators](/langsmith/evaluation-concepts#online-evaluation) that run on traces in real time or [offline evaluators](/langsmith/evaluation-concepts#offline-evaluation) that run on a dataset.
+<Callout icon="split" color="#8b5cf6" iconType="solid">
+LangSmith supports two types of evaluations based on **what objects they run on**:
 
-## Datasets
+- <Icon icon="flask" iconType="solid" /> [Offline evaluations](#offline-evaluation) run on _examples_ from _datasets_. Examples include reference outputs for comparison. Use offline evaluations for pre-deployment testing, benchmarking, and regression testing.
+- <Icon icon="radar" iconType="solid" /> [Online evaluations](#online-evaluation) run on _runs_ or _threads_ from [tracing](/langsmith/observability-quickstart) projects. These are real production traces without reference outputs. Use online evaluations for production monitoring and real-time performance tracking.
 
-A dataset is a collection of examples used for evaluating an application. An example is a test input, reference output pair.
+**The key distinction:** offline evaluations have access to reference outputs (what you expect), while online evaluations only have access to actual inputs and outputs (what actually happened).
+</Callout>
+
+## Evaluation lifecycle
+
+As you develop and [deploy your application](/langsmith/deployments), your evaluation strategy evolves from pre-deployment testing to production monitoring. LLM applications progress through distinct phases, each requiring different evaluation approaches. During development and testing, offline evaluations validate functionality against curated datasets. After deployment, online evaluations monitor production behavior on live traffic. As applications mature, both evaluation types work together in an iterative feedback loop to improve quality continuously.
+
+```mermaid
+graph LR
+    A[Development] --> B[Testing]
+    B --> C[Deployment]
+    C --> D[Monitoring]
+    D --> E[Iteration]
+
+    A -.-> F[Offline]
+    B -.-> F
+    C -.-> G[Online]
+    D -.-> G
+    E -.-> H[Both]
+
+    style F fill:#8b5cf6,stroke:#7c3aed,color:#fff
+    style G fill:#8b5cf6,stroke:#7c3aed,color:#fff
+    style H fill:#8b5cf6,stroke:#7c3aed,color:#fff
+```
+
+### 1. Development with offline evaluation
+
+Before production deployment, use offline evaluations to validate functionality, benchmark different approaches, and build confidence.
+
+1. Create a [dataset](/langsmith/manage-datasets) with representative test cases.
```

Comment:
I like this flow but I'd remove the numbered bullets for each of these sections. I think for someone new since they're super high level it makes the bullets hard to follow